locales/en/badge.json

```
@@ -1,12 +1,12 @@
 {
   "title": {
-    "connectWallet": "You must connect wallet first",
-    "mintable": "You can mint this badge with",
-    "notMintable": "You canont mint this badge. to Get this badge you need to met certain criteria",
-    "freeMintable": "You can mint this badge free",
-    "minted": "🎉🎉🎉 You have Badge already !! "
+    "connectWallet": "You must connect to a wallet first!",
```

You made me think. Does one connect the site to one's wallet, or does one connect one's wallet to the site?

The latter sounds more natural to me, since the wallet is what the user is metaphorically holding.

Suggestion:

```suggestion
    "connectWallet": "You must connect a wallet first!",
```